src/test/kotlin/no/nav/syfo/cronjob/behandlendeenhet/PersonBehandlendeEnhetCronjobSpek.kt

```diff
@@ -86,7 +108,7 @@ object PersonBehandlendeEnhetCronjobSpek : Spek({
                     )
                 }
 
-                it("should not update Enhet of existing PersonOversiktStatus if motebehovUbehandlet, moteplanleggerUbehandlet and oppfolgingsplanLPSBistandUbehandlet are not true)") {
+                it("should not update Enhet of existing PersonOversiktStatus if motebehovUbehandlet, moteplanleggerUbehandlet, oppfolgingsplanLPSBistandUbehandlet or dialogmotekandidat are not true)") {
```

Comment:
Denne var tricky 🤔 Er tanken at man skal ikke oppdatere enhet hvis alle er `not true`? Altså at man skal oppdatere hvis minst én er true?

Reply:
Blir vel `and` i testenavnet og ikke `or` fortsatt? Det er som tilsvarer at alle ikke er true.

Reply:
Stemmer det, var litt rask med renamingen der.